src/tile.cpp

```diff
@@ -1590,7 +1578,7 @@ void Tile::resetTileFlags(const Item* item)
 		resetFlag(TILESTATE_BED);
 	}
 
-	const Container* container = item->getContainer();
+	auto container = item->getContainer();
```

Comment:
Could this be `const auto`
```suggestion
	const auto container = item->getContainer();
```